src/loader/endpoints/get_inloco_states.py

```diff
@@ -2,12 +2,26 @@
 from utils import secrets
 from endpoints.helpers import allow_local
 import endpoints.GoogleDocsLib as gd
+import os
+import numpy as np
+
+configs_path = os.path.join(os.path.dirname(__file__), "aux")
```

Comment:
maybe adding this path to the `helpers.py` might be interesting. Then you don't need to instantiate it every time.

you'd be able to just import it

`from endpoints.helpers import allow_local, configs_path`